packages/xpath/test/helpers.ts

```diff
@@ -10,14 +10,33 @@ type AnyParentNode =
 	| XMLDocument;
 
 declare global {
+	/**
+	 * The timezone identifier used for all date and time operations in tests.
+	 * This string follows the IANA Time Zone Database format (e.g., 'America/Phoenix').
+	 * It determines the offset and DST behavior for `Date` objects and
+	 * related functions.
+	 *
+	 * @example 'America/Phoenix' // Fixed UTC-7, no DST
+	 * @example 'Europe/London' // UTC+0 (GMT) or UTC+1 (BST) with DST
+	 */
 	// eslint-disable-next-line no-var
 	var TZ: string | undefined;
 	// eslint-disable-next-line no-var
+	var LOCALE_ID: string | undefined;
+	/**
+	 * The locale string defining the language and regional formatting for tests.
+	 * This follows the BCP 47 language tag format (e.g., 'en-US'). It ensures consistent formatting
+	 * across tests.
+	 *
+	 * @example 'en-US' // American English
+	 */
+	// eslint-disable-next-line no-var
 	var IMPLEMENTATION: string | undefined;
```

Comment:
I didn't find the usage of this `IMPLEMENTATION`

Reply:
It was added [in this commit ](https://github.com/getodk/web-forms/commit/0d7c7bcf65113396fb7c4eb42c701865b76f421d#diff-afd1a869e5caed19e5fb890a10ad2875720fc73f6c46cd0c0f0fa9745a8c3816). It's unclear what it is for. I'll leave it there for now and ask Trevor later